### PR TITLE
Add Prometheus helm chart

### PR DESCRIPTION
**Pull request by ivyostosh**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/prometheus/prometheus/helm/patches/0002-image-patch.diff [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+diff --git a/charts/prometheus-server/templates/server/deploy.yaml b/charts/prometheus-server/templates/server/deploy.yaml
+index 3f327376..bbf8ce29 100644
+--- a/charts/prometheus-server/templates/server/deploy.yaml
++++ b/charts/prometheus-server/templates/server/deploy.yaml
+@@ -53,7 +53,7 @@ spec:
+       {{- end }}
+       containers:
+         - name: {{ template "prometheus.name" . }}-{{ .Values.server.name }}
+-          image: "{{ .Values.server.image.repository }}:{{ .Values.server.image.tag }}"
++          image: "{{ .Values.sourceRegistry }}/{{ .Values.server.image.repository }}@{{ .Values.server.image.digest }}"
```
Comment (TerryHowe) [inline projects/prometheus/prometheus/helm/patches/0002-image-patch.diff:21]: This is fine, but I've been thinking we should perfect the helper template and attempt to upstream so it supports tag or digest

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: projects/prometheus/prometheus/helm/patches/0002-image-patch.diff (hunk 1 above)
Comment (ivyostosh) [reply projects/prometheus/prometheus/helm/patches/0002-image-patch.diff:21]: I created https://github.com/aws/eks-anywhere-packages/issues/616 to track this for now. Ideally, we should be doing this for all packages.

**Pull request merged**